### PR TITLE
table default colour change and add sorting option in filter

### PR DESCRIPTION
**Pull request by maurya-k**

### CODE REVIEW

File: src/main/resources/config/liquibase/changelog/20200701600000_add_sorting_in_pivot_table.xml [hunk 1]
```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet author="khushbu" id="200701600000">
+
+        <update tableName="property_types">
+            <column name="color_picker_default_value" value='787878' />
+            <where>id = 677</where>
```
Comment (ir73) [inline src/main/resources/config/liquibase/changelog/20200701600000_add_sorting_in_pivot_table.xml:7]: can it happen that this id is not 677 for the customer? Should we use some other property for comparison?

File: src/main/resources/config/liquibase/changelog/20200701600000_add_sorting_in_pivot_table.xml [hunk 2]
```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet author="khushbu" id="200701600000">
+
+        <update tableName="property_types">
+            <column name="color_picker_default_value" value='787878' />
```
Comment (ir73) [inline src/main/resources/config/liquibase/changelog/20200701600000_add_sorting_in_pivot_table.xml:6]: I am not sure single quotes is a valid xml syntax, might be worth changing to double quotes

File: src/main/resources/config/liquibase/changelog/20200701600000_add_sorting_in_pivot_table.xml [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+        <insert tableName="field_types_property_types">
+            <column name="field_type_id" value="98" />
+            <column name="property_type_id" value="686" />
+            <column name="sequence_number" value="8" />
+        </insert>
+
+
+        <insert tableName="field_types_property_types">
+            <column name="field_type_id" value="99" />
+            <column name="property_type_id" value="686" />
```
Comment (ir73) [inline src/main/resources/config/liquibase/changelog/20200701600000_add_sorting_in_pivot_table.xml:40]: Are you trying to reference `Sort` property type? This seems a bit risky if the id is not 686. Perhaps you would want to make a `select` first to get the ID first?

Comment (ir73) [reply src/main/resources/config/liquibase/changelog/20200701600000_add_sorting_in_pivot_table.xml:40]: I guess that might be okay if you are 100% sure the ID cannot be anything else, but looks quite risky

File: src/main/resources/config/liquibase/changelog/20200701600000_add_sorting_in_pivot_table.xml (hunk 1 above)
Comment (maurya-k) [reply src/main/resources/config/liquibase/changelog/20200701600000_add_sorting_in_pivot_table.xml:7]: no, 677 is the id of the property

File: src/main/resources/config/liquibase/changelog/20200701600000_add_sorting_in_pivot_table.xml (hunk 3 above)
Comment (maurya-k) [reply src/main/resources/config/liquibase/changelog/20200701600000_add_sorting_in_pivot_table.xml:40]: > Are you trying to reference `Sort` property type? This seems a bit risky if the id is not 686. Perhaps you would want to make a `select` first to get the ID first?

686 id is already instead using CSV file so its always there

**Pull request merged**